src/app/pages/vendor-details/vendor-details.component.scss

```
@@ -0,0 +1,70 @@
+
+.vendor-hero-stats {
+  min-width: 11rem;
+}
+
+.vendor-hero-actions {
+  overflow-x: auto;
+}
+
+.block_full {
+  @apply px-6 py-6 bg-secondary border-emerald-400 border border-solid rounded-lg flex flex-col gap-4 relative overflow-hidden;
+  width: 100%;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)